benches/water.rs

```diff
@@ -41,9 +41,10 @@ fn energy_ewald(bencher: &mut Bencher) {
 fn forces_ewald(bencher: &mut Bencher) {
     let system = utils::get_system("water");
     let ewald = get_ewald();
+    let mut forces = vec![Vector3D::zero(); system.size()];
```

Comment:
Yep, that's right. The code will be executed each time, but not taken in account for the loop timing.